test-data/flow_example/flow.ert

```diff
@@ -0,0 +1,19 @@
+QUEUE_SYSTEM LOCAL
+QUEUE_OPTION LOCAL MAX_RUNNING 4
+
+RUNPATH spe1_out/realization-<IENS>/iter-<ITER>
+
+NUM_REALIZATIONS 12
+
+ECLBASE SPE1
+REFCASE <CONFIG_PATH>/resources/refcase/SPE1
+
+SUMMARY WOPT:PROD WWPT:PROD WGPT:PROD WWIT:INJ
+
+OBS_CONFIG obs.txt
+
+GEN_KW FIELD_PROPERTIES field_properties.tmpl field_properties.json field_properties_priors
+
+FORWARD_MODEL TEMPLATE_RENDER(<INPUT_FILES>=parameters.json, <TEMPLATE_FILE>=<CONFIG_PATH>/resources/SPE1.DATA.jinja2, <OUTPUT_FILE>=SPE1.DATA)
```

Comment:
We are not using the `field_properties.json` here (the `TEMPLATE_RENDER`forward model takes parameters directly from `parameters.json`) , and since we have made the template argument for `GEN_KW` optional, we can just omit this part. Then the `test-data/flow_example/field_properties.tmpl`can also be deleted :slightly_smiling_face: 
```suggestion
GEN_KW FIELD_PROPERTIES field_properties_priors

FORWARD_MODEL TEMPLATE_RENDER(<INPUT_FILES>=parameters.json, <TEMPLATE_FILE>=<CONFIG_PATH>/resources/SPE1.DATA.jinja2, <OUTPUT_FILE>=SPE1.DATA)
```